src/h3/mod.rs

```diff
@@ -574,6 +582,11 @@ pub enum Event {
     /// Stream was closed,
     Finished,
 
+    /// Stream was reset.
+    ///
+    /// The associated data represents the error code received by the peer.
```

Comment:
This reads a bit weird, can we say "sent by the peer", or maybe just "The associated data represents the error code"

Reply:
Oh, right, "sent by the peer" makes a lot more sense than "received by the peer".
```suggestion
    /// The associated data represents the error code sent by the peer.
```